spec/restaurant_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject(:restaurant) { described_class.new(menu: menu) }
+  let(:menu) { double :menu, view: list }
+  let(:list) { [
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.